tokenserver-common/src/error.rs

```diff
@@ -262,3 +273,24 @@ impl From<TokenserverError> for HttpResponse {
         ResponseError::error_response(&inner)
     }
 }
+
+impl ReportableError for TokenserverError {
+    fn error_backtrace(&self) -> String {
+        format!("{:#?}", self.backtrace)
+    }
+
+    fn is_reportable(&self) -> bool {
```

Comment:
Can you rename this method while you're here? We've been meaning to change it everywhere (already done in contile) to make it clearer:

```suggestion
    fn is_sentry_event(&self) -> bool {
```

I think `ReportableError` still works as the trait name (`is_reportable` wasn't clear as to where it would be reported to, sentry or metrics, but the trait handles both)

Reply:
Can do 👍🏻